source/slang/slang-ir-insts.h

```diff
@@ -2734,6 +2735,14 @@ IRTargetSpecificDecoration* findBestTargetDecoration(
         IRInst*         val,
         CapabilityAtom  targetCapabilityAtom);
 
+inline IRTargetIntrinsicDecoration* findBestTargetIntrinsicDecoration(
+    IRInst* inInst,
+    CapabilitySet const& targetCaps)
+{
+    return as<IRTargetIntrinsicDecoration>(findBestTargetDecoration(inInst, targetCaps));
+}
```

Comment:
I wonder if the function name here is a little inaccurate to what the function does. Probably merits a comment describing what it means.

To me, this seems to mean "find the best target decoration for the given instruction on a target with the given capabilities, and *if* that is a target-intrinsic decoration, return it."